src/bedriftsmeny/Bedriftsmeny.tsx

```diff
@@ -1,30 +1,45 @@
-import React, { FunctionComponent } from 'react';
+import React, {FunctionComponent, useEffect, useState} from 'react';
 import { Innholdstittel } from 'nav-frontend-typografi';
-import Virksomhetsvelger, { VirksomhetsvelgerProps } from './Virksomhetsvelger/Virksomhetsvelger';
+import Virksomhetsvelger from './Virksomhetsvelger/Virksomhetsvelger';
 import { Organisasjon, JuridiskEnhetMedUnderEnheterArray } from './Organisasjon';
 import './bedriftsmeny.less';
 import { History } from 'history';
+import {byggOrganisasjonstre} from "./byggOrganisasjonsTre";
 
 interface EgneProps {
     sidetittel?: string;
-    organisasjonstre?: JuridiskEnhetMedUnderEnheterArray[];
+    organisasjoner?: Organisasjon[];
     history: History;
+    onOrganisasjonChange: (organisasjon: Organisasjon) => void;
 }
 
-type AlleProps = EgneProps & VirksomhetsvelgerProps;
+const Bedriftsmeny: FunctionComponent<EgneProps> = (props) => {
+    const { sidetittel = 'Arbeidsgiver' } = props;
+    const [organisasjonstre, setOrganisasjonstre] = useState<
+        JuridiskEnhetMedUnderEnheterArray[] | undefined
+        >(undefined);
 
-const Bedriftsmeny: FunctionComponent<AlleProps> = (props) => {
-    const { sidetittel = 'Arbeidsgiver', ...virksomhetsvelgerProps } = props;
+    useEffect(() => {
+        const byggTre = async (organisasjoner: Organisasjon[]) => {
+            const juridiskeenheterMedBarn: JuridiskEnhetMedUnderEnheterArray[] = await byggOrganisasjonstre(
```

Comment:
Kan vi gå for enten "barn" eller "underenheter"? Det er ganske komplisert logikk her, og da synes jeg det fort kan bli forvirrende når navngivningen ikke er konsekvent.